docs/user/resources/05-metricpipeline.md

```diff
@@ -72,29 +72,29 @@ For details, see the [MetricPipeline specification file](https://github.com/kyma
 | **input.&#x200b;istio**  | object | Istio input configures collection of Istio metrics from applications running in the Istio service mesh. |
 | **input.&#x200b;istio.&#x200b;diagnosticMetrics**  | object | DiagnosticMetrics configures collection of additional diagnostic metrics. The default is `false`. |
 | **input.&#x200b;istio.&#x200b;diagnosticMetrics.&#x200b;enabled**  | boolean | If enabled, diagnostic metrics are collected. The default is `false`. |
-| **input.&#x200b;istio.&#x200b;enabled**  | boolean | Enabled specifies that istio-proxy metrics are scraped from Pods that have the istio-proxy sidecar injected. The default is `false`. |
+| **input.&#x200b;istio.&#x200b;enabled**  | boolean | Enabled specifies if the 'istio' input is enabled, so that istio-proxy metrics are scraped from Pods that have the istio-proxy sidecar injected. The default is `false`. |
 | **input.&#x200b;istio.&#x200b;envoyMetrics**  | object | EnvoyMetrics enables the collection of additional Envoy metrics with prefix `envoy_`. The default is `false`. |
 | **input.&#x200b;istio.&#x200b;envoyMetrics.&#x200b;enabled**  | boolean | Enabled specifies that Envoy metrics with prefix `envoy_` are scraped additionally. The default is `false`. |
 | **input.&#x200b;istio.&#x200b;namespaces**  | object | Namespaces configures the namespaces for which the collection should be activated. By default, all namespaces including system namespaces are enabled. |
-| **input.&#x200b;istio.&#x200b;namespaces.&#x200b;exclude**  | \[\]string | Exclude signals from the specified Namespace names only. |
-| **input.&#x200b;istio.&#x200b;namespaces.&#x200b;include**  | \[\]string | Include signals from the specified Namespace names only. |
+| **input.&#x200b;istio.&#x200b;namespaces.&#x200b;exclude**  | \[\]string | Exclude telemetry data from the specified Namespace names only. By default all namespaces (except system namespaes dependent on input type). An exclude list cannot be specified together with an include list. |
+| **input.&#x200b;istio.&#x200b;namespaces.&#x200b;include**  | \[\]string | Include telemetry data from the specified Namespace names only. By default all namespaces (except system namespaes dependent on input type). An include list cannot be specified together with an exclude list. |
 | **input.&#x200b;otlp**  | object | OTLP input configures the push endpoint to receive metrics from an OTLP source. |
-| **input.&#x200b;otlp.&#x200b;disabled**  | boolean | If set to `true`, no push-based OTLP signals are collected. The default is `false`. |
+| **input.&#x200b;otlp.&#x200b;disabled**  | boolean | Disabled specifies if the 'otlp' is deactivated, so that no push-based OTLP signals are collected. The default is `false`. |
```

Comment:
```suggestion
| **input.&#x200b;otlp.&#x200b;disabled**  | boolean | Disabled specifies if the 'otlp' input is deactivated, so that no push-based OTLP signals are collected. The default is `false`. |
```